### PR TITLE
Blog post about 2.4.0 release

### PR DESCRIPTION
**Pull request by ABBAPOH**

### CODE REVIEW

File: content/blog/release-2.4.0.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+Note that the new provider only works with Conan version 2.5.0 and above; for Conan version 1.x
+please use the [ConanfileProbe](http://localhost/~abbapoh/qbs-io//docs/qml-qbsprobes-conanfileprobe/).
+
+For more details on the provider as well as how to use Conan only for installing dependencies,
+see the [provider](https://qbs.io//docs/qml-qbsmoduleproviders-conan/) page in the documentation.
+
+### Other changes
+* We've added new FlatBuffers modules, for [C](https://qbs.io//docs/qml-qbsmodules-flatbuf-c/)
+  and [C++](https://qbs.io//docs/qml-qbsmodules-flatbuf-cpp/) languages. Those modules require
```
Comment (ckandeler) [inline content/blog/release-2.4.0.md]: "These modules"

File: content/blog/release-2.4.0.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  runtime dependencies that can be installed using Conan module provider
+  ([QBS-1666](https://bugreports.qt.io/projects/QBS/issues/QBS-1666)).
+* Rules trying to create artifacts outside the build directory is now a hard error
+  ([QBS-1268](https://bugreports.qt.io/projects/QBS/issues/QBS-1268)).
+* More details are now printed when a command times out
+  ([QBS-1750](https://bugreports.qt.io/projects/QBS/issues/QBS-1750)).
+* Updated the bundled quickjs library.
+* The pkg-config based fallback provider was removed. That provider was not flexible enough and
+  didn't give users control over providers priorities. Also, that provider created files
+  even for non-present modules which was inful in case of misconfiguration since those files
```
Comment (ckandeler) [inline content/blog/release-2.4.0.md]: "inful"?

File: content/blog/release-2.4.0.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  had to be cneaned up manually.
```
Comment (ckandeler) [inline content/blog/release-2.4.0.md]: typo

File: content/blog/release-2.4.0.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+### Contribute
+If You are a happy user of Qbs, please tell others about it. But maybe you would
+like to contribute something. Everything that makes Qbs better is highly
+appreciated. Contributions may consist of reporting bugs or fixing them right
+away. But also new features are very welcome. Your patches will be automatically
+sanity-checked, built and verified on Linux, macOS and Windows by our CI bot.
+Get started with instructions in the [Qbs Wiki](https://wiki.qt.io/Qbs).
+
+Thanks to everybody who made the 2.3 release happen:
```
Comment (ckandeler) [inline content/blog/release-2.4.0.md]: 2.4

**Review by ckandeler — CHANGES_REQUESTED** [review]

File: content/blog/release-2.4.0.md (hunk 2 above)
Comment (ABBAPOH) [reply content/blog/release-2.4.0.md]: looks like I was distracted while rewording this sentence=)

File: content/blog/release-2.4.0.md [hunk 5]
```diff
@@ -74,8 +74,8 @@ see the [provider](https://qbs.io//docs/qml-qbsmoduleproviders-conan/) page in t
 * Updated the bundled quickjs library.
 * The pkg-config based fallback provider was removed. That provider was not flexible enough and
   didn't give users control over providers priorities. Also, that provider created files
-  even for non-present modules which was inful in case of misconfiguration since those files
-  had to be cneaned up manually.
+  even for non-present modules which also happens in case of misconfiguration. This was
+  inconvinient since those files had to be cleaned up manually.
```
Comment (ckandeler) [inline content/blog/release-2.4.0.md]: "inconvenient"

**Review by ckandeler — APPROVED** [review]

**Pull request merged**